README.md

```diff
@@ -85,10 +85,11 @@ Host in Beijing. Go to [demo](http://122.112.182.72:8080).
 Follow this [document](docs/en/guides/How-to-build.md).
 
 # Contact Us
-* Submit an [issue](https://github.com/apache/skywalking/issues)
```

Comment:
Why we need to remove this?

Reply:
-> Issue should be kept as bug report and feature discussion, rather than contact channel, so I remove it.

Reply:
Got it